cmd/boulder-wfe/main.go

```diff
@@ -77,13 +85,19 @@ func setupWFE(c config, logger blog.Logger, stats metrics.Scope, clk clock.Clock
 	sac := bgrpc.NewStorageAuthorityClient(sapb.NewStorageAuthorityClient(saConn))
 
 	var rns noncepb.NonceServiceClient
-	if c.WFE.NonceService != nil {
-		rnsConn, err := bgrpc.ClientSetup(c.WFE.NonceService, tlsConfig, clientMetrics, clk)
-		cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to Nonce service")
+	npm := map[string]noncepb.NonceServiceClient{}
+	if c.WFE.GetNonceService != nil {
+		rnsConn, err := bgrpc.ClientSetup(c.WFE.GetNonceService, tlsConfig, clientMetrics, clk)
+		cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to nonce service")
```

Comment:
nit: I think it would be helpful to distinguish the error message when the `bgrpc.ClientSetup` for a `GetNonceService` fails, vs the error message when the `bgrpc.ClientSetup` for a `RedeemNonceServices` fails.

```suggestion
		cmd.FailOnError(err, "Failed to load credentials and create gRPC connection to get-nonce service")
```